### PR TITLE
Update docs to use our js styles

### PR DESCRIPTION
**Pull request by Jammjammjamm**
Edited some examples so that they were consistent with our linting rules, as well as some other minor documentation edits.

### CODE REVIEW

File: lib/smart.js [hunk 1]
```diff
@@ -4,14 +4,20 @@ const smartOauthUrl = 'http://fhir-registry.smarthealthit.org/StructureDefinitio
...
+ * Get the SMART OAuth URIs from a Capability Statement
  *
  * @example
  *
- * oauthUris = authFromCapability(capabilityStatement);
+ * {
+ *   authorizeUrl,
+ *   tokenUrl,
+ *   registerUrl,
+ *   launchRegisterUrl,
```
Comment (Jammjammjamm) [inline lib/smart.js]: Is this correct? The SMART docs mention a `manage` url, but I don't see a `launch-register` url.
http://docs.smarthealthit.org/authorization/conformance-statement/

Comment (foxtree) [reply lib/smart.js]: a google "verbatim" search for "launchRegisterUrl" turns up 0 hits.

Comment (foxtree) [reply lib/smart.js]: ...same thing with a github "advanced" search, either `extension:rb launchRegisterUrl` or `extension:js launchRegisterUrl`

0 hits, except for  our own repo.

Comment (foxtree) [reply lib/smart.js]: ^ This should perhaps be its own distinct issue or Trello card; "research".

**Pull request merged**